craft_providers/multipass/multipass_provider.py

```diff
@@ -90,6 +90,7 @@ def launched_environment(
         base_configuration: base.Base,
         build_base: str,
         instance_name: str,
+        allow_unstable: bool = False,
```

Comment:
We should add a `:param:` documentation for this argument in the method docstring.

Reply:
I updated the docstrings!